assets/stylesheets/sections/post-editor.scss

```diff
@@ -15,6 +15,7 @@
 @import '../main';
 @import '../shared/forms';
 
+@import './media';
```

Comment:
The problem with importing the `media` section here is that it increases the size of the `post-editor` section proportionally, going from 145KB to 168KB. One way to address that problem would be to update our framework to actually support multiple [CSS bundles](](https://github.com/Automattic/wp-calypso/blob/master/client/wordpress-com.js#L233-L240)), e.g.:

```javascript
sections.push( {
  name: 'post-editor',
  paths: [ '/post', '/page', '/edit' ],
  module: 'post-editor',
  group: 'editor',
  secondary: true,
  css: ['post-editor', 'media']
} );
```

That's not something easy, but totally doable.